lib/support/phone.rb

```diff
@@ -3,6 +3,12 @@ class Phone
 
     attr_accessor :country, :number
 
+    CODE_LENGTH_RANGE_BY_COUNTRY_CODE = {
```

Comment:
not sure we need this forward-looking planning structure if all of them are the same

`SHORT_CODE_LENGTH_RANGE = 5..6`